ateliermecanique-ws/src/main/java/com/champlain/ateliermecaniquews/customeraccountsmanagementsubdomain/businesslayer/CustomerAccountServiceImpl.java

```diff
@@ -75,5 +78,23 @@ public void deleteCustomerAccountByCustomerId(String customerId) {
         }
     }
 
+    @Override
+    public CustomerAccountResponseModel createCustomerAccountForoAuth(CustomerAccountoAuthRequestModel customerAccountoAuthRequestModel) {
+       CustomerAccount account = customerAccountRepository.save(customerAccountoAuthRequestMapper.requestModelToEntity(customerAccountoAuthRequestModel));
+        if(account==null){
+            //throw exception
+        }
+        return customerAccountResponseMapper.entityToResponseModel(account);
+    }

```

Comment:
W programmer. Implemented a significant amount for the app with this PR. Only nit pick it to throw exception in this method